pkgs/tools/misc/vdirsyncer/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib, fetchurl, pythonPackages }:
+
+pythonPackages.buildPythonPackage rec {
+  version = "0.4.2";
+  name = "vdirsyncer-${version}";
+  namePrefix = "";
+
+  src = fetchurl {
+    url = https://github.com/untitaker/vdirsyncer/archive/0.4.2.tar.gz;
+    sha256 = "008181mglbrp5jsvpxr59b6w4mw26h4s4gwij152i47mfbrizsl4";
+  };
+
+  pythonPath = with pythonPackages; [
+                icalendar
+                click
+                requests
+                lxml
+                setuptools
+                requests_toolbelt
+                requests2
+                atomicwrites
+
+                #pytest
+                #werkzeug
+                #radicale
+
+                # From AUR:
+                # python-click
+                # python-icalendar
+                # python-lxml
+                # python-requests-toolbelt
+                # python-setuptools
+                # python-pytest (check)
+                # python-pytest-localserver (check)
+                # python-pytest-xprocess (check)
+                # python-werkzeug-git (check)
+                # python-wsgi-intercept (check)
+                # radicale (check)
+                ];
+
+  buildInputs = [ ];
```

Comment:
It's [empty by default](https://github.com/NixOS/nixpkgs/blob/73c43b4e4f9d47e4a47fef6b210fb97a1ad8cc75/pkgs/development/python-modules/generic/default.nix#L13), so you can just leave it out.